packages/buckets/src/index.ts

```diff
@@ -1,25 +1,24 @@
-export * from './api'
-export * from './buckets'
-export * from './utils'
-
 // Exports all API response types for typescript users
 export {
-  Root,
-  LinksResponse,
-  ListIpfsPathResponse,
-  RemovePathResponse,
-  RemoveResponse,
-  RootResponse,
   ArchiveInfoResponse,
   ArchiveResponse,
   ArchiveStatusResponse,
   ArchiveWatchResponse,
   CreateResponse,
-  ListPathItem,
+  LinksResponse,
+  ListIpfsPathResponse,
   ListPathResponse,
   ListResponse,
+  PathItem,
```

Comment:
chuck in the PathMetadata

Reply:
oop, i meant MetadataObject

Reply:
gotcha

Reply:
metadata object is already exported from ./api

Reply:
and metadata (singular) comes from buckets_pb, so I think we're good on both fronts

Reply:
will `import { MetadataObject } from "@textile/hub"` work?

Reply:
ahhhh, sorry, my bad. of course

Reply:
reading the wrong lines. merge away